recipes/genome-properties/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "2.0.1" %}
+
+package:
+  name: genome-properties
+  version: {{ version }}
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('genome-properties', max_pin="x.x") }}
+
+source:
+  url: https://github.com/EBI-Metagenomics/genome-properties/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 628428e7ed2b74c16703fcf8e825e84d4cbcbadf4dc61dac6c64e89f35694ecc
+
+requirements:
+  build:
+    - make
+    - bioconda::perl-bioperl
+    - perl-app-cpanminus
```

Comment:
Those should not be in the build dependencies.

Reply:
yeah, these are only there to install the two perl packages with `cpanm`, so if that problem is sorted I could get rid of it